cartographer/cloud/internal/map_builder_server.h

```diff
@@ -64,10 +64,15 @@ class MapBuilderContext : public MapBuilderContextInterface {
                                   const std::string& sensor_id,
                                   const mapping::proto::LocalSlamResultData&
                                       local_slam_result_data) override;
+  void RegisterClientIdForTrajectory(int trajectory_id,
```

Comment:
consistency with function name: client_id, trajectory_id (if you want)

Reply:
done